iOS_SDK/OneSignalSDK/Source/OneSignalTracker.m

```diff
@@ -81,6 +82,10 @@ + (void)endBackgroundFocusTask {
 }
 
 + (void)onFocus:(BOOL)toBackground {
+    // Privacy consent can be set by remote params, need to wait until they are set
```

Comment:
Simplify comment?
`// Privacy consent is set through remote params request`

Reply:
given that we need the delay task to have this check, I removed it